### DISCUSSION TITLE
Vercel: Prepare Git Repository Error

### DISCUSSION THREAD

**Discussion by NalCol** [Qexo/Qexo#306] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/58576200/214769278-04542edc-5bc9-4a0e-92b9-057a93750be3.png)
这是为啥，搞不了了

**Comment by am-abudu** [comment]
请稍后重试。Vercel方面经常出现问题。

**Reply by NalCol** [reply]
昨天到今天一直这样

**Reply by am-abudu** [reply]
我这里刚测试了一下是正常的。我建议你尝试在 [该界面](https://vercel.com/account/login-connections) 重新绑定 Github 账户并重试。

**Reply by NalCol** [reply]
重新绑了，一样问题😂

**Reply by NalCol** [reply]
请问您开了proxy或者public仓库吗

**Reply by am-abudu** [reply]
如果是这样的话，你可以尝试将源码拉到本地之后使用 [Vercel-CLI](https://vercel.com/docs/cli) 进行部署

**Reply by am-abudu** [reply]
我使用了 Proxy 并使用 Private 仓库

**Reply by NalCol** [reply]
好

**Reply by NalCol** [reply]
看这个😂问题应该就出在这里
幸好这个是本地js运行的，容易debug
![image](https://user-images.githubusercontent.com/58576200/214784125-f0523202-5247-431d-af46-bf9c1472564c.png)

**Reply by NalCol** [reply]
还真是，看来Keep Email Private不可开😂

**Comment by NalCol** [comment, marked as answer]
**不能**开启 https://github.com/settings/emails 里的 Keep my email addresses private 选项
**DO NOT** enable "Keep my email addresses private" option in https://github.com/settings/emails
![image](https://user-images.githubusercontent.com/58576200/214785046-006387b0-4e69-4dd0-bd16-bfcca2cd3a61.png)
